test/saithrift/tests/saiacl.py

```diff
@@ -1729,6 +1732,9 @@ def runTest(self):
 
         sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr_subnet1, ip_mask, nhop1)
         sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr_subnet2, ip_mask, nhop2)
+        sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr1_subnet, ip_mask, rif_id1)
+        sai_thrift_create_route(self.client, vr_id, addr_family, ip_addr2_subnet, ip_mask, rif_id2)
```

Comment:
Thanks.
Does 'ip_mask' in line 1736 mean 'ip_mask2'?
Please remove the local routes in cleanup stage.

Reply:
Updated review comments ,cleanup section